spec/dummy/config/database.yml

```diff
@@ -1,15 +1,14 @@
 defaults: &defaults
-  host: localhost
+  host: <%= ENV.fetch('THREDDED_DB_1_PORT_5432_TCP_ADDR', 'localhost') %>
```

Comment:
So, @shanesveller, does this mean pointing to `db` here, theoretically would work?

The one snag that jumps out at me would be that, outside of using Docker, I would have to define `db` in my own host `/etc/hosts` file ... whereas here it looks for that ENV and falls back to localhost, which we know everyone has (pretty much).